app/gui/view/component-browser/searcher-list-panel/src/column_grid.rs

```diff
@@ -0,0 +1,189 @@
+//! Wrapper around multiple `component_group::View` that provides a layout where the
+//! `component_group::View` are stacked in three columns. Designed for use in the sections of a
+//! Component Browser Panel.
+
+use ensogl_core::display::shape::*;
+use ensogl_core::prelude::*;
+
+use enso_frp as frp;
+use ensogl_core::application::frp::API;
+use ensogl_core::application::Application;
+use ensogl_core::data::color;
+use ensogl_core::define_endpoints_2;
+use ensogl_core::display;
+use ensogl_core::display::object::ObjectOps;
+use ensogl_gui_component::component;
+use ensogl_list_view as list_view;
+use ide_view_component_group as component_group;
+use ide_view_component_group::Layers;
+use ordered_float::OrderedFloat;
+
+
+
+// ==========================
+// === Constants + Colors ===
+// ==========================
+
+const COLUMN_GAP: f32 = 2.0;
+
+const BASE_COLORS: &[color::Rgba] = &[
+    color::Rgba::new(0.527, 0.554, 0.18, 1.0),
+    color::Rgba::new(43.0 / 255.0, 117.0 / 255.0, 239.0 / 255.0, 1.0),
+    color::Rgba::new(62.0 / 255.0, 139.0 / 255.0, 41.0 / 255.0, 1.0),
+    color::Rgba::new(192.0 / 255.0, 71.0 / 255.0, 171.0 / 255.0, 1.0),
+];
+
```

Comment:
1. What are these colors? "Base Colors" don't tell me much.
2. Why are they not in the theme?